### DISCUSSION TITLE
是否可以webdav支持自签的SSL证书  ,在选项中提供忽略ssl安全警告

### DISCUSSION THREAD

**Discussion by 1103394945** [remotely-save/remotely-save#715] [Ideas]
是否可以webdav支持自签的SSL证书  ,在选项中提供忽略ssl安全警告

**Comment by fyears** [comment]
你的 webdav 是怎样搭建的呢

**Reply by fyears** [reply]
地址是内网 ip？类似 `192.168.xxx.xxx` 这样的吗？我试了一下，最新版本可以直接连 http 的喔？

**Reply by 1103394945** [reply]
原本是我的自建nas中提供的webdav   但我的nas使用的是自签的ssl证书   不过最近已经申请了正常的ssl证书  可以正常使用插件了

**Reply by 1103394945** [reply]
感谢你的回复
